examples/experimental/onnx_ptq_classification.py

```diff
@@ -0,0 +1,58 @@
+import os
```

Comment:
I'd put it in the `examples/experimental/onnx` folder
otherwise experimental nas sample for torch will be mixed up with the onnx one